### PR TITLE
DRAFT: release pipelines configurations

### PR DESCRIPTION
**Pull request by awsluja**
#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] [Pull request labels](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#labels) are added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: shared-scripts.sh [hunk 1]
```diff
@@ -275,3 +288,55 @@ function _scanArtifacts {
...
+}
+function _deploy {
+    loadCache repo $CODEBUILD_SRC_DIR
+    loadCache all-binaries $CODEBUILD_SRC_DIR/out
+
+    source .circleci/local_publish_helpers.sh
+    _loadS3AccountCredentials
+    uploadPkgCli
+
+    ./out/amplify-pkg-linux-x64 --version
```
Comment (sdstolworthy) [inline shared-scripts.sh:328]: This is probably debug code? Does this need to be here?

File: codebuild_specs/release_workflows/hotfix_workflow.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
+version: 0.2
+env:
+  shell: bash
+  compute-type: BUILD_GENERAL1_MEDIUM
+  variables:
+    AWS_DEFAULT_REGION: us-east-1
+    AWS_REGION: us-east-1
+    CDK_DEFAULT_REGION: us-east-1
+    CLI_REGION: us-east-1
+    TEST_SUITE: src/__tests__/auth_2b.test.ts
```
Comment (sdstolworthy) [inline codebuild_specs/release_workflows/hotfix_workflow.yml:10]: Do all the `TEST_SUITE` vars need to be removed?

**Review by sdstolworthy — CHANGES_REQUESTED** [review]

**Comment by sobolk** [comment]
Please incorporate publish.sh script split, see for reference https://github.com/aws-amplify/amplify-cli/pull/12775 .

**Comment by awsluja** [comment]
Closing in favor of: https://github.com/aws-amplify/amplify-cli/pull/12918

**Pull request closed without merging**